### PR TITLE
Kk 382 Immunization

### PR DESCRIPTION
**Pull request by issyzac**
Restrict the app from asking about already captured vaccinations in previous visits

### CODE REVIEW

File: opensrp-chw/src/main/java/org/smartregister/chw/application/ChwSyncConfiguration.java [hunk 1]
```diff
@@ -60,8 +60,10 @@ public boolean isSyncSettings() {
     @Override
     public SyncFilter getEncryptionParam() {
         return SyncFilter.PROVIDER;
+        //return SyncFilter.TEAM_ID;
```
Comment (gosso22) [inline opensrp-chw/src/main/java/org/smartregister/chw/application/ChwSyncConfiguration.java:63]: Can we remove this?

**Review by gosso22 — CHANGES_REQUESTED** [review]
Can we remove the comment on EncryptionParam?

File: opensrp-chw/src/main/java/org/smartregister/chw/application/ChwSyncConfiguration.java (hunk 1 above)
Comment (issyzac) [reply opensrp-chw/src/main/java/org/smartregister/chw/application/ChwSyncConfiguration.java]: Sure, removed!

**Review by gosso22 — APPROVED** [review]

**Comment by gosso22** [comment]
LGTM - This is a great implementation!

**Pull request merged**